packages/model-viewer/src/three-components/Renderer.ts

```diff
@@ -141,21 +145,28 @@ export class Renderer extends EventDispatcher {
       this.threeRenderer = new WebGLRenderer({
         canvas: this.canvas3D,
         alpha: true,
-        antialias: true,
+        antialias: false,
```

Comment:
Is this necessary?

Reply:
I saw many examples using it so I set it for the beginning, but I would like to do some tests to see if it's really needed. 
If it is then the solution is to use the default anti aliasing when no post processing is enabled, otherwise use the anti-alias pass with the post processing pipeline

Reply:
You can't enable/disable alpha on the fly, nor per MV element, so I hope we can keep this `true`.

Reply:
According to the pmndrs wiki:

*The default WebGL antialiasing solution has no effect in post processing pipelines because it only applies to the canvas. It's recommended to set the <code>antialias</code> flag to <code>false</code> when creating a <code>WebGLRenderer</code> for a post processing setup.*

So the answer is we can keep it enabled, but it would be a possible performance benefit if it could be disabled when postprocessing is running. Since there is no way to dynamically enable/disable it, the solution is to keep it on permanently.